localstack/utils/aws/templating.py

```diff
@@ -76,13 +92,11 @@ def apply(obj, **_):
             rendered_template = json.loads(rendered_template)
         return rendered_template
 
-    def prepare_namespace(self, variables) -> Dict:
-        context_var = variables.get("context") or {}
-        stage_var = variables.get("stage_variables") or {}
-        namespace = {
-            "context": context_var,
-            "stageVariables": stage_var,
```

Comment:
Moving `stageVariables` to `VelocityUtilApiGateway`, as this is API GW specific and does not belong into the common base class.